.github/workflows/format.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       run: |
         isort --diff --check --profile black -l 88 -m 3 \
             sky/skylet/providers/ibm/
-    - name: Running isort for yapf formatted files
+    - name: Running isort for ruff formatted files
```

Comment:
Ruff has isort capabilities as well so this step can ideally be merged with the above

Reply:
Do you know how to configure it in the ruff settings?

Reply:
Sorry for the late reply. Here's the documentation to add it as a pylint rule - https://docs.astral.sh/ruff/rules/#isort-i with example config - https://docs.astral.sh/ruff/faq/#how-does-ruffs-import-sorting-compare-to-isort